docs/general/staking-dashboard.md

```diff
@@ -0,0 +1,250 @@
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Ledger app installed, or a browser
+extension such as the [Polkadot-JS Extension](./polkadotjs.md#polkadot-js-extension) or
+[other third-party extensions](./wallets.md#browser-extensions) with a funded
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account.
+
+On top of the [existential deposit](../learn/learn-accounts.md#existential-deposit-and-reaping), you
+need some free balance to pay for [transaction fees](../learn/learn-transaction-fees.md) and the
+minimum amount to place your nominations (only for nominators, currently
+{{ polkadot: __<RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={2500000000000} filter="humanReadable"/>__ :polkadot }}{{ kusama: __<RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>__ :kusama }})
+or join a nomination pool (only for pool members, currently
+{{ polkadot: __<RPC network="polkadot" path="query.nominationPools.minJoinBond" defaultValue={10000000000} filter="humanReadable"/>__ :polkadot }}{{ kusama: __<RPC network="kusama" path="query.nominationPools.minJoinBond" defaultValue={1667000000} filter="humanReadable"/>__ :kusama }}).
+For more information about staking on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
+visit the [staking page](../learn/learn-staking.md) and the
+[advanced staking page](../learn/learn-staking-advanced.md).
+
+:::
+
+The [Polkadot Staking Dashboard](https://staking.polkadot.network/#/overview) is a tool only
+dedicated to [staking](../learn/learn-staking.md) on Polkadot,
+[Kusama](../learn/learn-kusama-vs-polkadot.md) (Polkadot's canary network) and
+[Westend](../maintain/maintain-networks.md#westend-test-network) (Polkadot's test network).
+
+The dashboard is not a wallet, meaning that you cannot transfer funds between accounts. To fund
+accounts, you can use [third-party wallets](./wallets.md) or the
+[Polkadot-JS UI](./polkadotjs-ui.md). You can participate in staking by being a
+[nominator](../learn/learn-nominator.md) or a member of a
+[nomination pool](../learn/learn-nomination-pools.md). Once you have a funded account, you need to
+connect it to the dashboard (connect button on the top-right corner), this will allow you to
+interact with {{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} native staking.
+
+:::info The dashboard is a Web3 dApp
+
+The dashboard is a decentralized application (dApp), meaning that under the hood there are no
+centralized servers but a network of nodes storing information about the Polkadot blockchain and
+validating transactions, including everything you decide to do on the dashboard. To login into the
+dashboard you do not need emails and passwords but just your public address. From now on, you are in
+charge of your data and will need to authorize anything you want to do. Welcome to Web3!
```

Comment:
This is true, assuming the user is connected to the light client. The RPC servers connected to may still be run by centralized entities. I'm removing the ambiguity here.

```suggestion
The dashboard is a decentralized application (dApp) and to login, you do not need to sign up with an email and password, but just an account created on the supported non-custodial wallets. Any transaction to be submitted, needs to be signed by you. Also, if you use the light client which is resistant to censorship, you are interacting trustlessly with the network without any intermediaries. Welcome to the world of true Web3!
```